compiler/src/dmd/cparse.d

```diff
@@ -1890,6 +1890,14 @@ final class CParser(AST) : Parser!AST
                 if (specifier.alignExps)
                     error("no alignment-specifier for typedef declaration"); // C11 6.7.5-2
 
+                if (specifier.vector_size)
+                {
+                    auto length = new AST.IntegerExp(token.loc, specifier.vector_size / dt.size(), AST.Type.tuns32);
+                    auto tsa = new AST.TypeSArray(dt, length);
+                    dt = new AST.TypeVector(tsa);
+                    specifier.vector_size = 0;          // used it up
```

Comment:
Why is this needed?

Reply:
To fix the first problem mentioned in https://issues.dlang.org/show_bug.cgi?id=24125#c2